DSharpPlus/Clients/DiscordClient.Dispatch.cs

```diff
@@ -671,20 +673,26 @@ internal async Task OnReadyEventAsync(ReadyPayload ready, JArray rawGuilds, JArr
             this.guilds[guild.Id] = guild;
         }
 
-        await this.ready.InvokeAsync(this, new SessionReadyEventArgs());
+        await this.events[typeof(SessionCreatedEventArgs)]
+            .As<SessionCreatedEventArgs>()
+            .InvokeAsync(this, new SessionCreatedEventArgs());

```

Comment:
So... every time a gateway event is received, a dictionary lookup is required with a type cast?

Reply:
See https://discord.com/channels/379378609942560770/379386901725052928/1245887242824847422